css/bootstrap5/privatebin.css

```diff
@@ -56,6 +56,10 @@
 	transition: background-color 0.75s ease-out;
 }
 
+.dropdown-menu {
+	--bs-dropdown-min-width: 23rem;
```

Comment:
BTW you can also put all variables in `:root` , where it is all applicable: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#using_the_root_pseudo-class

(Just FYI)

Reply:
Thanks, that is good to know. I think in this case it is less likely to have side effects to target only that class. Although the name suggests it relates only to dropdowns.